Parser/asdl_c.py

```diff
@@ -1132,13 +1132,93 @@ def visitModule(self, mod):
     return result;
 }
 
+/* copy.replace() support (shallow copy) */
+static PyObject *
+ast_type_replace(PyObject *self, PyObject *args, PyObject *kwargs) {
+    if (!_PyArg_NoPositional("__replace__", args)) {
+        return NULL;
+    }
+
+    struct ast_state *state = get_ast_state();
+    if (state == NULL) {
+        return NULL;
+    }
+
+    PyObject *result = NULL;
+    // known AST class fields
+    PyObject *fields = NULL;
+    // current instance dictionary
+    PyObject *__dict__ = NULL;
```

Comment:
It is better to not use names with double underscores in C. They are reserved for compiler. Use just "dict".